bin/gen-crons.py

```diff
@@ -23,6 +23,8 @@ def main():
                             'Only define for cron.d style crontabs.'))
     parser.add_option('-p', '--python', default='python2.6',
                       help='Python interpreter to use.')
+    parser.add_option('-n', '--nologdir', action='store_true',
```

Comment:
Are you sure of how to inject it into the real environment? Should be able to do it in global.conf I think. That likely would be cleaner to only do the log stuff if a `LOG_DIR` is defined.